Indexing/ClauseCodeTree.cpp

```diff
@@ -590,7 +579,7 @@ Clause* ClauseCodeTree::ClauseMatcher::next(int& resolvedQueryLit)
   }
 
   for(;;) {
-    LiteralMatcher* lm=lms.top();
+    Recycled<LiteralMatcher, NoReset>& lm = lms.top();
 
```

Comment:
Is there an advantage of using `Recycled&` here. If we just want to use the object, the `&*` could be used, right?

Reply:
Yes exactly! Doesn't make any difference here.